tgstation.dme

```diff
@@ -9,7 +9,6 @@
 // END_FILE_DIR
 
 // BEGIN_PREFERENCES
-#define DEBUG

```

Comment:
erhm yea... I switched debbuging in DreamMaker off in hope the lag would go away and didn't reenable it.